docs/snapshot.md

```diff
@@ -74,16 +65,30 @@ This looks like the following:
 
 ![tarball plugins screenshot][7]
 
-### /meta
+### /podlogs
 
-The `/meta` directory contains metadata about this Sonobuoy run, including configuration and query runtime.
+The `/podlogs` directory contains logs for each pod found during the Sonobuoy run, similarly to what you would get with `kubectl logs -n <namespace> <pod> <container>`.
 
-- `/meta/query-time.json` - Contains metadata about how long each query took, example: `{"queryobj":"Pods","time":12.345ms"}`
-- `/meta/config.json` - A copy of the Sonobuoy configuration that was set up when this run was created, but with unspecified values filled in with explicit defaults, and with a `UUID` field in the root JSON, set to a randomly generated UUID created for that Sonobuoy run.
+- `/podlogs/<namespace>/<podname>/<containername>.log` - Contains the logs for the each container, for each pod in each namespace.
 
 This looks like the following:
 
-![tarball meta screenshot][8]
+![tarball podlogs screenshot][6]
+
+### /resources
+
+The `/resources` directory contains JSON-serialized Kubernetes objects, taken from querying the Kubernetes REST API. The directory has the following structure:
+
+- `/resources/ns/<namespace>/<type>.json` - For all resources that belong to a namespace, where `<namespace>` is the namespace of that resource (eg. `kube-system`), and `<type>` is the type of resource, pluralized (eg. `Pods`).
+- `/resources/cluster/<type>.json` - For all resources that don't belong to a namespace, where `<type>` is the type of resource, pluralized (eg. `Nodes`).
+
+This looks like the following:
+
+![tarball resources screenshot][4]
+
+### /servergroups.json
+
+`/servergroups.json` contains a collection of Kubernetes APIs that the cluster supports.
```

Comment:
This is slightly odd wording -- in context it's understandable, but "contains a collection of Kubernetes APIs" sounds like the APIs themselves. You mean something more like "lists the Kubernetes APIs that ...", yes?
The "<thing> contains" pattern doesn't need to hold for a file description. (I realize that it does for `serverversion.json`, but it's a different critter, too, and could probably use a description rewrite if I ever have time to spare.)

Reply:
I'm approving anyway -- it's not a required change and if my fingers continue to twitch, I can fix up when/if I go in and edit more systematically.